lighthouse-core/test/computed/page-dependency-graph-test.js

```diff
@@ -9,8 +9,8 @@ const PageDependencyGraph = require('../../computed/page-dependency-graph.js');
 const BaseNode = require('../../lib/dependency-graph/base-node.js');
 const NetworkRequest = require('../../lib/network-request.js');
 
-const sampleTrace = require('../fixtures/traces/progressive-app-m60.json');
-const sampleDevtoolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
+const sampleTrace = require('../fixtures/traces/progressive-app-m79.json');
```

Comment:
should I have generate this trace/logs in a different way than just running on `https://pwa.rocks/` w/ default options? A lot of the numbers that changed with `-u` seem too different.

Reply:
the trace that was recorded before used devtools throttling, see my other thoughts on using a new trace though :)